packages/usa-memorable-date/src/usa-memorable-date.twig

```diff
@@ -1,18 +1,34 @@
 <fieldset class="usa-fieldset">
-  <legend class="usa-legend">Date of birth</legend>
-  <span class="usa-hint" id="dobHint">For example, 4 28 1986</span>
+  <legend class="usa-legend">Date of Birth</legend>
+  <span class="usa-hint" id="mdHint">For example: January 19 2000</span>
```

Comment:
Hint was hard to read with `01 - January 19 2000` — while that's accurate when it comes to the select contents, it's confusing. Using only the month name will show how that users need month / day / year without the unfamiliar formatting